test/gdb_script.py

```diff
@@ -69,16 +77,16 @@
     "help",
     "help reset",
     "help halt",
-    "help resume",
     "init",
     "reset",
     "reset halt",
-    "reset halt resume",
```

Comment:
why were these removed?

Reply:
The "resume" monitor commands were disabled back in 2015.